lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -125,12 +132,56 @@ defmodule ExUnit.CLIFormatter do
     Map.update(test_counter, type, 1, &(&1 + 1))
   end
 
+  ## Slowest
+
+  defp format_slowest_total(%{slowest: slowest} = config, run_us) do
+    slowest_us =
+      config
+      |> extract_slowest_tests()
+      |> Enum.reduce(0, & &1.time + &2)
+
+    slowest_time =
+      slowest_us
+      |> normalize_us()
+      |> format_us()
+
+    percentage = Float.round(((slowest_us / run_us) * 100), 1)
+
+    "Top #{slowest} slowest tests (#{slowest_time}s), #{percentage}% of total time:\n"
+  end
+
+  defp format_slowest_times(config) do
+    config
+    |> extract_slowest_tests()
+    |> Enum.map(&format_slow_test/1)
+  end
+
+  defp format_slow_test(%ExUnit.Test{name: name, time: time}) do
+    "  * #{name} (#{format_us(time)}ms)\n"
+  end
+
+  defp extract_slowest_tests(%{slowest: slowest, test_timings: timings} = _config) do
+    timings
+    |> Enum.sort_by(& -&1.time)
```

Comment:
I find the capture syntax readable, but I appreciate that it may seem obfuscated to others. I've replaced it with an anonymous function.